Yosemite/YosemiteTests/Stores/MediaStoreTests.swift

```diff
@@ -181,7 +181,38 @@ final class MediaStoreTests: XCTestCase {
         XCTAssertEqual(remote.invocations, [.loadMediaLibrary(siteID: sampleSiteID)])
     }
 
-    /// Verifies that `MediaAction.retrieveMediaLibrary` from a JCP site returns the media list from the remote response.
+    /// Verifies that `MediaAction.retrieveMediaLibrary` for a placeholder site ID returns the media list from the remote response.
+    func test_retrieveMediaLibrary_returns_media_list_when_connecting_to_site_with_placeholder_site_id() throws {
+        // Given
+        let siteID = WooConstants.placeholderSiteID
+        let remote = MockMediaRemote()
+        let media = WordPressMedia.fake()
+        remote.whenLoadingMediaLibraryFromWordPressSite(siteID: siteID, thenReturn: .success([media]))
+        let mediaStore = MediaStore(dispatcher: dispatcher,
+                                    storageManager: storageManager,
+                                    network: network,
+                                    remote: remote)
+
+        insertJCPSiteToStorage(siteID: sampleSiteID)
```

Comment:
We want to test with placeholder site ID in this case, so I think we don't need to insert JCP site to storage here?

Reply:
Done as part of 801eb4a7c9ec8c0365269733052e675f8e7f95f7